test/web-platform-tests/run-single-wpt.js

```diff
@@ -162,12 +181,8 @@ function createJSDOM(urlPrefix, testPath, expectFail) {
           };
 
           window.add_result_callback(test => {
-            if (test.status === 1) {
-              errors.push(`Failed in "${test.name}": \n${test.message}\n\n${test.stack}`);
-            } else if (test.status === 2) {
-              errors.push(`Timeout in "${test.name}": \n${test.message}\n\n${test.stack}`);
-            } else if (test.status === 3) {
-              errors.push(`Uncompleted test "${test.name}": \n${test.message}\n\n${test.stack}`);
+            if (test.status === test.FAIL || test.status === test.TIMEOUT || test.status === test.NOTRUN) {
```

Comment:
I think we can just do `test.status !== test.PASS`

Reply:
There is the `PRECONDITION_FAILED` status, which is reported when a normative optional feature is not implemented.